front/components/PlansTables.tsx

```diff
@@ -390,3 +399,80 @@ export function PricePlans({
     );
   }
 }
+
+export function ProPricePlans({
```

Comment:
Can you remove all the styles from sparkles 🙏

Reply:
Ooopsie 😅